README.adoc

```diff
@@ -8,11 +8,6 @@ GitHub repo for SteeltoeOSS Docker images.
 
 == Building
 
-.Build all images (not yet implemented)
-----
-$ build.ps1 -All
-----
-
 .Build a specific image
 ----
 $ build.ps1 config-server
```

Comment:
Running the instruction below fails for me:
```
manifest-tool --username * --password * push from-spec config-server/manifest.yml
manifest-tool: The term 'manifest-tool' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

Reply:
After running `./build.ps1 spring-boot-admin`, I would have expected an image named "steeltoeoss/spring-boot-admin" to appear, but instead it creates "steeltoeoss/spring-boot-admin-amd64-linux". Is that intentional?

Reply:
> The $ prefix gets in the way when copying this line. And PowerShell requires `./` for running a script from the current directory.

Agreed. There have been some cases in Steeltoe (that I'm pretty sure I saw last week) where that's used to separate a command from its output, but there's got to be a better way.

> For Steeltoe users unfamiliar with docker...

As far as general instruction goes, I agree it could be better... I was planning to come back through (when we are 100% sure of which docker images we want to be shipping) during the connecting-the-dots efforts after either the milestone/preview or GA of v4 and make sure all of the metadata is in good shape both in the repo and on docker hub.

> After running `./build.ps1 spring-boot-admin`...

I was attempting to avoid having any impact on the images declared out of scope. The process for SBA should be a match for what it was before I touched anything, which does include the extras on the end. There may have been some other (manual?) process that produced the shorter form name without the os info

Reply:
I can't find any real benefit to using manifests right now, I stand by the decision to remove it (until/unless some reason comes up in the future)